panimg/image_builders/dicom.py

```diff
@@ -111,6 +111,10 @@ def _iter_origins(self):
                 else:
                     yield np.array(file_origin, dtype=float)
 
+    @staticmethod
+    def _invert_intensities(array: np.ndarray) -> np.ndarray:
+        return -array + array.max() + array.min()
```

Comment:
Isn't `-array` dangerous if the value is -128 in a int8 array, for example?

An option could be to convert the array to float if the values need to be inverted - that is also the behavior when any kind of rescaling is applied.

Reply:
Nevermind, seems to work fine!